tests/rest/client/test_account.py

```diff
@@ -17,16 +15,22 @@
 import os
 import re
 from email.parser import Parser
-from typing import Optional
+from typing import Dict, List, Optional
+from unittest.mock import Mock
 
 import pkg_resources
 
+from twisted.internet.testing import MemoryReactor
```

Comment:
To make the olddeps CI happy:
```suggestion
from twisted.test.proto_helpers import MemoryReactor
```